channeldb/models/channel_edge_policy.go

```diff
@@ -198,3 +198,57 @@ func (c *ChannelEdgePolicy1) Sig() (input.Signature, error) {
 // A compile-time check to ensure that ChannelEdgePolicy1 implements the
 // ChannelEdgePolicy interface.
 var _ ChannelEdgePolicy = (*ChannelEdgePolicy1)(nil)
+
+type ChannelEdgePolicy2 struct {
+	lnwire.ChannelUpdate2
```

Comment:
Any particular reason why we're using composition here, instead of creating new attributes for the fields we care about (potentially with more concrete types)?

Reply:
Ah I guess we can just re-use the existing serialization as well? Potentially yet another reason to TLV everywhere for the wire messages.